src/features/game/components/Decorations.tsx

```diff
@@ -274,6 +307,34 @@ export const Decorations: React.FC = () => {
           alt="Beaver"
         />
       )}
+
+      {state.inventory["Bird Bath"] && (
```

Comment:
I think you should use `<Bath />` component similar to how `<Scarecrows />` . else it's not used anywhere in the file

Reply:
thanks, fixed it.
about `yarn tsc` and `yarn format` after I run it all the files are marked as changed and there are 180 warnings, even on files that I didn't touch

Reply:
Hello @JacobSobolev 
Those warnings must be shown because of difference in Unix vs Windows endline characters formatting.
No need to worry about them. :)